docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -1,29 +1,39 @@
 ---
 id: eks-terraform
-title: "Deploy an EKS cluster with Terraform"
+title: "Deploy an EKS cluster with Terraform (advanced)"
 description: "Deploy an Amazon Kubernetes Cluster (EKS) with a Terraform module for a quick Camunda 8 setup."
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 This guide offers a detailed tutorial for deploying an Amazon Web Services (AWS) Elastic Kubernetes Service (EKS) cluster, tailored explicitly for deploying Camunda 8 and using Terraform, a popular Infrastructure as Code (IaC) tool.
 
-This is designed to help leverage the power of IaC to streamline and reproduce a Cloud infrastructure setup. By walking through the essentials of setting up an Amazon EKS cluster, configuring AWS IAM permissions, and integrating a PostgreSQL database, this guide explains the process of using Terraform with AWS, making it accessible even to those new to Terraform or IaC concepts.
+It is recommended to use this guide for building a robust and sustainable infrastructure over time. However, for a quicker trial or proof of concept, using the [eksctl](./eksctl.md) method may suffice.
+
+This guide is designed to help leverage the power of Infrastructure as Code (IaC) to streamline and reproduce a Cloud infrastructure setup. By walking through the essentials of setting up an Amazon EKS cluster, configuring AWS IAM permissions, and integrating a PostgreSQL database and an OpenSearch domain (as an alternative to ElasticSearch), this guide explains how to use Terraform with AWS, making it accessible even to those new to Terraform or IaC concepts. It utilizes AWS-managed services when available, providing these as an optional convenience that you can choose to use or not.
 
 :::tip
 
 If you are completely new to Terraform and the idea of IaC, read through the [Terraform IaC documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code) and give their [interactive quick start](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code#quick-start) a try for a basic understanding.
 
 :::
 
-## Prerequisites
+## 0. Introduction
+
+### Requirements
```

Comment:
just noticed AWS CLI is missing since it's being used for minimum the S3 bucket creation.

Reply:
and terraform auth

Reply:
yes yes, I'm just saying the AWS CLI is missing in the requirements since it's used by terraform and for the s3 bucket.